modules/i18n/src/translations/en-US/portals/custom-auth-connection.ts

```diff
@@ -0,0 +1,144 @@
+                },
+                label: "Select the authentication type you are implementing",
+                title: "Authentication Type",
+                twoFactorAuthenticationCard: {
+                    examples: "E.g., TOTP",
+                    header: "2FA Authentication",
+                    mainDescription: "Only verify users in a second or later step in the login flow."
+                }
+            },
+            configurationsStep: {
+                authenticationTypeDropdown: {
+                    authProperties: {
+                        accessToken: {
+                            label: "Access token",
+                            placeholder: "Access Token",
+                            validations: {
+                                required: "Access Token is a required field."
+                            }
+                        },
+                        header: {
+                            label: "Header",
+                            placeholder: "Header",
+                            validations: {
+                                invalid: "Please choose a valid header name that adheres to the given guidelines.",
+                                required: "Header is a required field."
+                            }
+                        },
+                        password: {
+                            label: "Password",
+                            placeholder: "Password",
+                            validations: {
+                                required: "Password is a required field."
+                            }
+                        },
+                        username: {
+                            label: "Username",
+                            placeholder: "Username",
+                            validations: {
+                                required: "Username is a required field."
+                            }
+                        },
+                        value: {
+                            label: "Value",
+                            placeholder: "Value",
+                            validations: {
+                                required: "Value is a required field."
+                            }
+                        }
+                    },
+                    hint: "Once added, these secrets will not be displayed. You will only be able to reset them.",
+                    label: "Authentication Scheme",
+                    placeholder: "Select Authentication Type",
+                    title: "Endpoint Authentication",
+                    validations: {
+                        required: "Authentication Type is a required field."
+                    }
+                },
+                endpoint: {
+                    hint: "The URL of the configured external endpoint to integrate with the authenticator",
+                    label: "Endpoint",
+                    placeholder: "https://abc.external.authenticator/authenticate",
```

Comment:
Shall we add a meaning placeholder? for example, in branding ui input we have used `https://myapp.io` etc.

```suggestion
                    placeholder: "https://abc.external.authenticator/authenticate",
```

Reply:
These has been agreed upon prior to starting the development.

**Email Subject**: 
- [Architecture] Facilitate user authentication via a external service (custom authentication extension)